airgun/entities/host_new.py

```diff
@@ -949,6 +950,16 @@ def remediate_with_insights(
         view = RemediationView(self.browser)
         view.remediate.click()
 
+    def get_host_facts(self, entity_name):
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        self.browser.plugin.ensure_page_safe()
+        view.wait_displayed()
+        self.browser.wait_for_element(view.dropdown, exception=False)
+        assert view.dropdown.has_item('Facts')
```

Comment:
These are not required, anyways it will fail if it is not able to see the option of  `Facts`